Ka-Block/macOS/Base.lproj/Main.storyboard

```diff
@@ -45,9 +45,8 @@
         <scene sceneID="R2V-B0-nI4">
             <objects>
                 <windowController id="B8D-0N-5wS" sceneMemberID="viewController">
-                    <window key="window" title="Window" allowsToolTipsWhenApplicationIsInactive="NO" autorecalculatesKeyViewLoop="NO" oneShot="NO" releasedWhenClosed="NO" showsToolbarButton="NO" visibleAtLaunch="NO" animationBehavior="default" id="IQv-IB-iLA">
-                        <windowStyleMask key="styleMask" titled="YES" closable="YES" miniaturizable="YES" resizable="YES"/>
```

Comment:
I think we should bring back the `miniaturizable="YES"` yellow icon even though it's a little silly. There's something about seeing the yellow icon that makes the spotlight menu more recognizable.

Even silly little built in apps keep it.

<img width="486" alt="screen shot 2017-11-25 at 10 37 28 pm" src="https://user-images.githubusercontent.com/137/33237830-933e260a-d233-11e7-81fc-e99fe3445d5a.png">